### PR TITLE
MaryiaF/refactor: subscribe proposal_open_contract globally without s…

### PR DESCRIPTION
**Pull request by maryiafrantsava-binary**
…pecific contract_id, use same approach as in Deriv

During open contact mode runtime subscribe request happens with proposal_open_contract globally, without specific contract_id.

### CODE REVIEW

File: src/javascript/_common/base/socket_base.js [hunk 1]
```diff
@@ -180,9 +180,6 @@ const BinarySocketBase = (() => {
 
         if (isReady() && !availability.is_down && config.isOnline()) {
             is_disconnect_called = false;
-            if (!getPropertyValue(data, 'passthrough') && !getPropertyValue(data, 'verify_email')) {
-                data.passthrough = {};
```
Comment (maryiafrantsava-binary) [inline src/javascript/_common/base/socket_base.js:184]: I suppose we don’t need to pass an empty object in the request, as in Deriv-app

**Review by akmal-binary — APPROVED** [review]

**Review by matin-binary — APPROVED** [review]

File: src/javascript/app/pages/trade/purchase.js [hunk 2]
```diff
@@ -225,7 +225,6 @@ const Purchase = (() => {
         if (show_chart) {
             const request = {
                 proposal_open_contract: 1,
-                contract_id           : receipt.contract_id,
```
Comment (balakrishna-binary) [inline src/javascript/app/pages/trade/purchase.js:228]: global subscription should be made only one time. It should not be done for each purchase of contract

Comment (maryiafrantsava-binary) [reply src/javascript/app/pages/trade/purchase.js:228]: @balakrishna-binary thank you, let me fix it👌

Comment (maryiafrantsava-binary) [reply src/javascript/app/pages/trade/purchase.js:228]: @balakrishna-binary I've fixed

**Comment by matin-binary** [comment]
Closing this PR as the changes are not needed. 
`proposal_open_contract ` needs to be global only in Portfolio page which is already being handled.
On dashboard, there is only a single purchase at a time so no need for global `proposal_open_contract`.

**Pull request closed without merging**